core-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -77,7 +77,8 @@ clouddebug.stashbuttontext=Stash local changes
 clouddebug.attach=Attach
 clouddebug.nologin=You must be logged in to perform this action.
 clouddebug.noprojectid=Please enter a Project ID.
-clouddebug.nomodule=Please select a Module.
+clouddebug.nomoduleselected=Please select a Module.
```

Comment:
Anyone know the UI standards for capitalization? seems weird to have "Module" here and "module" in the very next line.